server/services/auditService.ts

```diff
@@ -0,0 +1,29 @@
+import HmppsAuditClient, { AuditEvent } from '../data/hmppsAuditClient'
+
+export enum Page {
+  EXAMPLE_PAGE = 'EXAMPLE_PAGE',
+}
+
+export interface PageViewEventDetails {
+  who: string
+  subjectId?: string
+  subjectType?: string
+  correlationId?: string
+  details?: object
+}
+
+export default class AuditService {
+  constructor(private readonly hmppsAuditClient: HmppsAuditClient) {}
+
+  async logAuditEvent(event: AuditEvent) {
+    await this.hmppsAuditClient.sendMessage(event)
+  }
+
+  async logPageView(page: Page, eventDetails: PageViewEventDetails) {
```

Comment:
This method might not belong in the template. The thinking was that most services will probably want to audit page views and this provides them a way to do that out of the box.